__tests__/commands/install/integration.js

```diff
@@ -437,6 +437,24 @@ test.concurrent('install with file: protocol as default', (): Promise<void> => {
   });
 });
 
+test.concurrent('install file: dedupe dependencies', async (): Promise<void> => {
+  await runInstall({}, 'install-file-dedupe-dependencies', async (config, reporter) => {
+    expect(await fs.exists(path.join(config.cwd, 'node_modules', 'a', 'node_modules'))).toEqual(false);
```

Comment:
Add a comment above this stating what you are testing or be more explicit with your test, please. I'm guessing this is to make sure `b` is not added as a sub-dependency of `a`, right?